### ISSUE TITLE
[Fix] cat_trigger_bodyshot not as reliable as cat_aimbot_hitboxmode

### ISSUE THREAD

**Issue by SylveonBottle** [nullifiedcat/cathook#142]
We had a chat about this before, Onee-chan. 

With cat_aimbot_slow 1, and triggerbot set to Head (with bodyshots if low enough health), the aimbot will attempt to lock on to the chest at a certain health but the triggerbot will not shoot because it does not believe the shot will kill.

This results in you staring at someone without shooting, and having to either manually look up to the head or manually shoot.

The best example of this is aiming at another sniper with stock rifle. Aimbot (autoshoot=0 slowaim=1 hitboxmode=auto-head) will stare at their chest because it can one-shot kill them, but triggerbot doesn't know this and won't try to bodyshot.

tl;dr throw code from aimbot auto-head into triggerbot bodyshot

**Comment by julianacat** [comment]
fixed in #158
